### DISCUSSION TITLE
What young 🍼 animal is called an elver?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1097] [Galaxy Trivia] [answered]
**Bonus:** Which 🐤 is also known as the South American ostrich?

**Comment by peterbaccaro** [comment]
The **baby eel** is called an elver.

The **pterocnemia pennata** is also known as the South America Ostrich. It is grey-brown with white spots. Its long neck and its beak are of a hazel shade.

**Comment by sheri-na** [comment]
A young eel is called an elver and rheas are known as the South American ostrich.

**Comment by thuongtruong1009** [comment]
A young 🍼 animal called an elver is a young eel, specifically a young European eel.

The 🐤 bird that is also known as the South American ostrich is the rhea. There are two species of rhea: the greater rhea (Rhea americana) and the lesser rhea (Rhea pennata). These flightless birds are native to South America and are often referred to as "ostriches" due to their similar appearance and habitat, even though they belong to a different family.

**Comment by ParanoidUser** [comment]
**Round 2** 
What animal can see the most colors?
What animal can jump the highest relative to its body size?

**Reply by peterbaccaro** [reply, marked as answer]
**Mantis shrimp** are known to have one of the most complex and elaborate visual systems among animals. While humans have three types of colour receptors (cones) in their eyes, which allows us to see a wide range of colours, mantis shrimp can have up to 16 types of colour receptors. This means they can perceive an extensive spectrum of colours, including ultraviolet and polarized light.

The animal that can jump the highest relative to its body size is the **flea**. Fleas are incredibly efficient jumpers and can leap up to 150 times their own body length, which is an impressive feat considering their small size. This ability is attributed to their powerful leg muscles and a unique elastic protein in their exoskeleton that allows them to store and release energy rapidly during a jump. This enables fleas to achieve remarkable heights and distances compared to their tiny bodies.

**Comment by sheri-na** [comment]
The animal that can see the most colors is the mantis shrimp, they can detect ten times more colors than a human.
A klipspringer can jump the highest relative to its own size, it can jump 10 times their own height.
